Makefile

```
@@ -56,19 +75,24 @@ protoc-plugins:
 	@go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2@$(PROTOC_GEN_OPENAPIV2_VERSION)
 .PHONY: protoc-plugins
 
-all: install build
+all: build
 .PHONY: all
 
 test:
+ifeq ($(UNAME_OS),Darwin)
+	# Workaround for PATH not working on macOS
+	$(BIN_DIR)/gotestsum -- -timeout 5m -p 1 ./...
+else
 	gotestsum -- -timeout 5m -p 1 ./...
+endif
```

the workaround wouldn't be needed if you just did:
```
GOTESTSUM := $(BIN_DIR)/gotestsum
```

And: 

```suggestion
	$(GOTESTSUM) -- -timeout 5m -p 1 ./...
```